### ISSUE TITLE
Remaining work for API Completeness

### ISSUE THREAD

**Issue by HowardWolosky** [PowerShell/PowerShellForGitHub#70]
Using a single issue to track the remaining work, so as not to litter the Issues and give the wrong impression on the current state of the project, bug wise.

If a contributor is interested in making progress towards any of these items, they should just create a new Issue to track that specific work, mention this Issue, and then resolve their PR against the Issue that they created.  At that point, I'll go ahead and update this list and check off that item.

[ ] Activity
   [ ] [Core][Activity](https://developer.github.com/v3/activity/)
   [ ] [Events](https://developer.github.com/v3/activity/events/) (Issue Events are already supported)
   [ ] [Feeds](https://developer.github.com/v3/activity/feeds/)
   [ ] [Notifications](https://developer.github.com/v3/activity/notifications/)
   [ ] [Starring](https://developer.github.com/v3/activity/starring/)
   [ ] [Watching](https://developer.github.com/v3/activity/watching/)
[ ] Gists
   [ ] [Core](https://developer.github.com/v3/gists/)
   [ ] [Comments](https://developer.github.com/v3/gists/comments/)
[ ] Git Data
   [ ] [Blobs](https://developer.github.com/v3/git/blobs/)
   [ ] [Commits](https://developer.github.com/v3/git/commits/)
   [ ] [Refs](https://developer.github.com/v3/git/refs/)
   [ ] [Tags](https://developer.github.com/v3/git/tags/)
   [ ] [Trees](https://developer.github.com/v3/git/trees/)
[ ] Migrations
   [ ] [Orgs](https://developer.github.com/v3/migrations/orgs/)
   [ ] [Source Imports](https://developer.github.com/v3/migrations/source_imports/)
   [ ] [Users](https://developer.github.com/v3/migrations/users/)
[ ] Organizations
   [ ] [Core](https://developer.github.com/v3/orgs/) (Some of this work already exists)
   [ ] [Blocking Users](https://developer.github.com/v3/orgs/blocking/)
   [ ] [Blocking Users](https://developer.github.com/v3/orgs/blocking/)
   [ ] [Members](https://developer.github.com/v3/orgs/members/)
   [ ] [Outside Collaborators](https://developer.github.com/v3/orgs/outside_collaborators/)
   [ ] [Webhooks](https://developer.github.com/v3/orgs/hooks/)
[ ] Projects
   [ ] [Core](https://developer.github.com/v3/projects/)
   [ ] [Cards](https://developer.github.com/v3/projects/cards/)
   [ ] [Collaborators](https://developer.github.com/v3/projects/collaborators/)
   [ ] [Columns](https://developer.github.com/v3/projects/columns/)
[ ] Pull Requests
   [ ] [Core](https://developer.github.com/v3/pulls/)
   [ ] [Reviews](https://developer.github.com/v3/pulls/reviews/)
   [ ] [Review Comments](https://developer.github.com/v3/pulls/comments/)
   [ ] [Review Requests](https://developer.github.com/v3/pulls/review_requests/)

> TODO: Complete this list for Reations, Repositories, Search and Teams

**Comment by HowardWolosky** [comment]
@etgottli added [Milestone](https://developer.github.com/v3/issues/milestones/) support for Issues with #62

**Comment by HowardWolosky** [comment]
@joseartrivera added [Label](https://developer.github.com/v3/issues/labels/) support for Issues with #59

**Comment by raghav710** [comment]
I would like to contribute to this. How do I find out whether someone has already taken up an issue? Is that status tracked somewhere?

**Comment by HowardWolosky** [comment]
Awesome.  We'd love to have you.  Going forward, all status is being tracked right here on GitHub.  If there's something that you'd like to tackle, please go ahead and create a separate Issue tracking that specific item, and assign it to yourself, then start working on it.  I thought that given the quantity of items in the checklist above, I didn't want to create individual Issues until we knew they were being covered by someone.

**Comment by raghav710** [comment]
@HowardWolosky  I would like to propose that we add a "Create/Update/Delete" branch utility. Branches are currently mentioned under [refs](https://developer.github.com/v3/git/refs/) which I believe isn't intuitive to the user. Thoughts?

**Comment by HowardWolosky** [comment]
@raghav710  -- The list above is simply tracking the API set that remains to be implemented, not necessarily indicating what the function names would be.  For refs, I would imagine that the functions would use API-based names (e.g. `Get-GitHubReference`, `Create-GitHubReference`, etc...) but those would also be aliased-out to names that might make more sense to users, as you suggest (like `Get-GitHubBranch`, `Create-GitHubBranch`, etc...)

**Comment by HowardWolosky** [comment]
#110 now tracking Releases support.

**Comment by themilanfan** [comment]
I'll help contribute when I have some free time 😄

**Comment by roboaks** [comment]
Any idea when the Teams | Members feature might emerge? I am specifically looking for the ability to add users to teams. Alas, I am relatively new to PowerShell and have no time to work on this anyway :-(

**Comment by HowardWolosky** [comment]
> Any idea when the Teams | Members feature might emerge?

There is no specific schedule for any new functionality coming in.  This is primarily community driven through contributions.

The only features that I know are specifically on the horizon is a large expansion of the pull request functionality.  That will likely be getting merged in before the end of the year.

**Comment by vercellone** [comment]
I'm considering contributing GitHubCodespaces.ps1, although I don't see codespaces in the list above.   I've started with Get-GitHubCodespace in my [feature/GitHubCodespaces](https://github.com/microsoft/PowerShellForGitHub/compare/master...vercellone:PowerShellForGitHub:feature/GitHubCodespaces) branch.   Should I proceed and create a new issue?

**Comment by HowardWolosky** [comment]
> I'm considering contributing GitHubCodespaces.ps1, although I don't see codespaces in the list above. I've started with Get-GitHubCodespace in my [feature/GitHubCodespaces](https://github.com/microsoft/PowerShellForGitHub/compare/master...vercellone:PowerShellForGitHub:feature/GitHubCodespaces) branch. Should I proceed and create a new issue?

I don't believe [CodeSpaces](https://docs.github.com/en/rest/codespaces) existed at the time that this issue was first created.  If you're interested in working on it, by all means please do create a tracking issue and proceed.  Thanks!
